Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/resource_printer_test.go

```diff
@@ -670,7 +773,7 @@ func TestPrintHunmanReadableIngressWithColumnLabels(t *testing.T) {
 		},
 	}
 	buff := bytes.Buffer{}
-	printIngress(&ingress, &buff, PrintOptions{false, false, false, false, false, false, []string{"app_name"}})
+	printIngress(&ingress, &buff, PrintOptions{false, false, false, false, false, false, false, "", []string{"app_name"}})
```

Comment:
I'd suggest:

```go
printIngress(&ingress, &buff, PrintOptions{ColumnLabels: []string{"app_name"}})
```

Most times it's a bad idea to initialize a big struct like that with positional fields instead of explicit field names.
With so many boolean fields, if somebody would reorder or add more fields (like we're doing) code like that would compile, while potentially changing the semantics (a field that used to be initialized to false accidentally becoming true and so on).